docs/supported-versions.asciidoc

```diff
@@ -2,6 +2,7 @@
 * OpenShift 4.11-4.14
 * Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), and Amazon Elastic Kubernetes Service (EKS)
 * Helm: 3.2.0+
+* VMware Tanzu Kubernetes Grid Versions: 	1.4, 1.5.4
```

Comment:
I am not sure we want to highlight here that we support a very old version of Tanzu K8s that is no longer supported by VMWare and that is based on a Kubernetes version that is no longer supported (1.22). If anything we should strike Tanzu from the support matrix.

Reply:
Morning @pebrc thanks for sharing !!! you are right, as I'm not sure whats of both it's the most important (I think Support Matrix becase our Support is based on it ...) I don't know what and were we have update it, I just want to share because some coworkers and I use this documentation in from our customers and I feel it's not so clear.